### DISCUSSION TITLE
Android app doesn't support button click to jump to a new window

### DISCUSSION THREAD

**Discussion by xihuaxixuegui** [beeware/briefcase#1678] [Q&A]
I run the app under the window, click the button to successfully jump to a new window, package and install it on the Android tablet, the same button clicks can't jump to the window, the error displays:W/python.stderr:   
File "/data/data/com.example.helloworld/files/chaquopy/AssetFinder/requirements/toga/handlers.py", line 80, in _handler
W/python.stderr:   File "/data/data/com.example.helloworld/files/chaquopy/AssetFinder/app/helloworld/app.py", line 130, in button_handler1
W/python.stderr:   File "/data/data/com.example.helloworld/files/chaquopy/AssetFinder/app/helloworld/app.py", line 34, in __init__
W/python.stderr:   File "/data/data/com.example.helloworld/files/chaquopy/AssetFinder/requirements/toga/window.py", line 182, in __init__
W/python.stderr:   File "/data/data/com.example.helloworld/files/chaquopy/AssetFinder/requirements/toga_android/window.py", line 39, in __init__
W/python.stderr: RuntimeError: Secondary windows cannot be created on mobile platforms

**Comment by freakboy3742** [comment]
The error message is telling you exactly what is going wrong: Android doesn't support creating secondary windows. 

It doesn't matter that this is legal on desktop platforms - Android and iOS don't support creating an instance of Window other than the main window for the app.
